spa/tsconfig.json

```diff
@@ -24,3 +23,6 @@
 	},
   "include": ["src"]
 }
+
+// FAIL src/pages/ConfigSteps/test.tsx
```

Comment:
Is this meant to be here?

Reply:
This was cleaned up it was not meant to be here